project/src/test/java/com/google/sps/servlets/QueryServletTest.java

```diff
@@ -54,9 +56,10 @@ public void setUp() throws Exception {
 
   @Test
   public void getRequest_fetchedMoreThanMaxNumPlaces_respondMaxNumPlaces() throws Exception {
+    setRequestParameters(/*cuisines*/ "sushi");
```

Comment:
Suggestion: add an overload to `setRequestParameters`, which does not accept any parameters. This will initialize a request with a default set ({"sushi"}).

Reply:
That's a great point. 
I decided to handle the whole thing a bit differently, in a way that I think is now better.

Reply:
Looks good!